tree/20_cfe_basics/ncf_lib.cf

```diff
@@ -772,3 +772,241 @@ body edit_defaults ncf_empty_select(select)
+      "svc_action_cmd[refresh]"   string => "/usr/bin/refresh -s ${service}";
+      "svc_action_cmd[reload]"    string => "${svc_action_cmd[refresh]}";
+      "svc_action_cmd[is-active]" string => "/usr/bin/lssrc -s ${service} | grep -q 'active'";
+      "action_command"            string => "${svc_action_cmd[${action}]}";
+      "method"                    string => "src";
+
+    # init.d
+    # WARN: Some actions may not be supported by the init script
+    pass1.!is_boot_action.!systemctl_utility_present.!svcadm_utility_present.!service_utility_present.!startsrc_utility_present.init_d_directory_present::
+      "action_command" string => "/etc/init.d/${service} ${action}";
+      "method"         string => "/etc/init.d/";
+
+    # windows
+    # Implementation is done in services promises
+    windows::
+      "method"         string => "Windows Service Manager";
+
+    ###########################################################################
+
+    any::
+      "canonified_service"        string => canonify("${service}");
+      "canonified_action"         string => canonify("${action}");
+      "canonified_action_command" string => canonify("${action_command}");
+
+      "old_class_prefix"  string => "ncf_services_${canonified_service}_${canonified_action}";
+      "promisers"          slist => { @{this.callers_promisers}, cf_null }, policy => "ifdefined";
+      "class_prefix"      string => canonify(join("_", "promisers"));
+      "args"               slist => { "${service}", "${action}" };
+
+  classes:
+
+    windows::
+      "is_valid_action"     or => {
+                                    strcmp("start", "${action}"),
+                                    strcmp("stop", "${action}"),
+                                    strcmp("restart", "${action}"),
+                                  };
+      "is_restart_action" expression => strcmp("restart", "${action}");
+
+    any::
+      "is_boot_action"      or => { 
+                                    strcmp("enable", "${action}"),
+                                    strcmp("disable", "${action}"),
+                                    strcmp("is-enabled", "${action}")
+                                  };
+
+      "is_check_action"     or => {
+                                    strcmp("is-active", "${action}"),
+                                    strcmp("is-active-process", "${action}"),
+                                    strcmp("is-enabled", "${action}")
+                                  };
+
+      "is_process_action" expression => strcmp("is-active-process", "${action}");
+
+    pass1::
+      "method_found"      expression => isvariable("method");
+
+    debian_8::
+      # This is to workaround a bug in Debian Jessie, that makes systemctl is-enabled
+      # fail if run against a service that still uses SysV style scripts
+      # See https://bugs.debian.org/cgi-bin/bugreport.cgi?bug=760616
```

Comment:
This bug was marked as fixed on 29/05/2015, is it still relevant to keep this exception ?